build.proj

```diff
@@ -8,12 +8,15 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <BuildNative Condition="'$(BuildNative)'==''">true</BuildNative>
+    <BuildTests Condition="'$(BuildTests)'==''">false</BuildTests>
     <BuildPackages Condition="'$(BuildPackages)'==''">true</BuildPackages>
   </PropertyGroup>
 
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
+    <Project Include="src\Native\build-native.proj" Condition="'$(BuildNative)'=='true'"/>
```

Comment:
Just curious why this happens out of the "normal" order?  Wouldn't it make more sense if `src\dirs.proj` would build this project, since this project is under the `src` directory?

Reply:
We can consider moving it to `src\dirs.proj` but it isn't a normal project and is controlled by a property from the root build so I thought it made more sense here.